### ISSUE TITLE
TS.WriteToConsole shows DB native units (metric)

### ISSUE THREAD

**Issue by Tich1212** [HydrologicEngineeringCenter/ExcelToCWMS#18]
![image](https://user-images.githubusercontent.com/57423836/117869530-c14ad700-b24f-11eb-8ac7-f27e314795bd.png)

```C#
 public void WriteToConsole()
        {
            Console.WriteLine("TSID = "+TSID);
            Console.WriteLine("Units = " + Units);
            foreach (var item in data)
            {
                Console.WriteLine("{0:dd-MMM-yyyy HHmm}{1,10:f3}{2,8:d}", item.Key, item.Value.Value, item.Value.Quality);
            }
        }
```

**Comment by Tich1212** [comment]
LookUpUnits is always returning null.

https://github.com/HydrologicEngineeringCenter/ExcelToCWMS/blob/5e8666f9c0553e61d4dd5752677bf9607c564631/Hec.Cwms/CwmsDatabase.cs#L70-L76

So ReadTimeSeries units are null and the timeseries data is return in metric:
https://github.com/HydrologicEngineeringCenter/ExcelToCWMS/blob/5e8666f9c0553e61d4dd5752677bf9607c564631/Hec.Cwms/CwmsDatabase.cs#L79-L86

**Comment by Tich1212** [comment]
Issue addressed with Pull Request #20.
